.vscode/deprecated-alias.code-snippets

```diff
@@ -0,0 +1,9 @@
+{
+	"Deprecated alias": {
```

Comment:
```suggestion
	"Deprecated alias for mathlib": {
```
Our two other code snippets explicitly mention mathlib, and I think this is a good policy.